README.md

```diff
@@ -2,10 +2,12 @@
 
 ## Prerequisites
 
-You need to have *ant*, *git* and *nodeJS* (version 4 or 5) installed.
+You need to have *ant*, *git* and *nodeJS* (version 10.0.0 or higher) installed.
 
 ### For Mac OS X
 
+**Note: Installation via homebrew is deprecated since the support for latest node.js versions has been stopped.** 
```

Comment:
Homebrew does support installation of node 10: `node@10`. I'll test to see if this works with the PR, and if so I'll add a commit noting this.

Reply:
(Looks like the hsg-setup wiki is [already updated](https://github.com/HistoryAtState/hsg-project/wiki/Setup/_compare/2ab4d2a43081e3c3b344df35483a78b6b6a89b4a...70ff2deff8c19958e6dea2bca0f00be188217fa4) to tell folks to use node@10... The corresponding changes in this PR are necessary for that updated set of instructions to work.)

FYI, Alexandra, here are the results of `brew install node@10`:

```
==> Downloading https://homebrew.bintray.com/bottles/node@10-10.16.0.mojave.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/7b/7b0ea520752c4b4577744b53af7a93b486b5a9d9081d71ab0a8289b2e6223e22?__gda__=exp=1563374276~hmac=951edd51bbcfc31123d9dadb333344693a18de01aed5ccbba00ccf9ca4ee2f30&response-content-di
######################################################################## 100.0%
==> Pouring node@10-10.16.0.mojave.bottle.tar.gz
==> Caveats
node@10 is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

If you need to have node@10 first in your PATH run:
  echo 'export PATH="/usr/local/opt/node@10/bin:$PATH"' >> ~/.bash_profile

For compilers to find node@10 you may need to set:
  export LDFLAGS="-L/usr/local/opt/node@10/lib"
  export CPPFLAGS="-I/usr/local/opt/node@10/include"

==> Summary
🍺  /usr/local/Cellar/node@10/10.16.0: 4,144 files, 49.4MB
```

Since we adopted node, the hsg-project setup wiki has always instructed us to install both `node` **and** a specific version of node - e.g., `node@4`, then `node@6`, now `node@10`. Without installing `node` proper, command line calls to `npm install -g bower` would fail because, as the version-specific installer notes:

> node@10 is keg-only, which means it was not symlinked into /usr/local, because this is an alternate version of another formula

(That other formula is `node`.)